referencia/webpayrest/README.md

```diff
@@ -667,9 +809,14 @@ Content-Type: application/json
  "details": [
      {
          "amount": 10000,
-         "commerce_code": Próximamente...,
+         "commerce_code": "Próximamente...",
```

Comment:
Isn't this the same commerce_code below?

Reply:
Fixed in f40d528, commerce code can't be visible, they shouldn't be displayed yet.